client/components/pack/AddPack.js

```diff
@@ -39,7 +39,7 @@ export const AddPack = () => {
    * @return {void}
    */
   const handleAddPack = () => {
-    dispatch(addPack({ name, owner_id: user?._id, is_public: isPublic }));
+    mutation.mutate({ name, owner_id: user?._id, is_public: isPublic })
```

Comment:
we should be returning mutation.mutate in the hook so we can call addPack like you did for edit pack item

Reply:
Ok i will push it again along with delete pack item